openadapt/config.py

```diff
@@ -100,6 +99,14 @@
     list(stop_str) for stop_str in STOP_STRS
 ] + SPECIAL_CHAR_STOP_SEQUENCES
 
+env_file = ".env"
```

Comment:
Please rename to ENV_FILE_PATH

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/223/commits/3992d1c78d334a352054c2575db248df8677ec4a